src/main/kotlin/no/skatteetaten/aurora/boober/service/DeployService.kt

```diff
@@ -79,11 +78,25 @@ class DeployService(
         val deployId = UUID.randomUUID().toString()
 
         val openShiftResponses: List<OpenShiftResponse> = performOpenShiftCommands(deployId, deploymentSpec)
-        val performDeploy = !(deploymentSpec.deploy?.flags?.pause ?: false) && shouldDeploy
-        val redeployResponse: ResponseEntity<JsonNode>? = if (performDeploy) triggerRedeploy(deploymentSpec, openShiftResponses) else null
 
         val success = openShiftResponses.all { it.success }
-        return AuroraDeployResult(deployId, deploymentSpec, openShiftResponses, redeployResponse, success)
+        val noPause = !(deploymentSpec.deploy?.flags?.pause ?: false)
+        val performDeploy = success && noPause && shouldDeploy
+
+        val tagResult = if (performDeploy) {
+            deploymentSpec.deploy?.takeIf { it.releaseTo != null }?.let {
+                val dockerGroup = it.groupId.replace(".", "_")
```

Comment:
This could very well be put into a utility method (i.e. createDockerGroupSafeName) to better communicate intent.